src/Mod/Arch/ArchSite.py

```diff
@@ -278,14 +278,14 @@ def Activated(self):
                 else:
                     warning = True
         if warning :
-            message = translate( "Arch" ,  "Please select only Building objects or nothing!\n\
-Site are not allowed to accept other object than Building.\n\
-Other objects will be removed from the selection.\n\
-You can change that in the preferences.") + "\n"
+            message = translate( "Arch" ,  "Please either select only Building objects or nothing at all!\n\
+                                            Site is not allowed to accept any other object besides Building.\n\
+                                            Other objects will be removed from the selection.\n\
+                                            Note: You can change that in the preferences." )
             ArchCommands.printMessage( message )
         if sel and len(siteobj) == 0:
             message = translate( "Arch" ,  "There is no valid object in the selection.\n\
-Site creation aborted.") + "\n"
+                                            Site creation aborted.") + "\n"
```

Comment:
is this correct formatting as well ?

Reply:
You cannot add all these lines if they weren't in the original text, otherwise it shifts everything in dialog boxes